lib/api/controllers/authController.js

```diff
@@ -84,6 +83,9 @@ module.exports = function AuthController (kuzzle) {
         delete response._source._id;
 
         return q(new ResponseObject(requestObject, response));
+      })
+      .then(response => {
```

Comment:
shouldn't we better : 
* 1st call the trigger on "response"
* then return the ResponseObject with the modified response
?

I mean:
    return kuzzle.repositories.user.load(requestObject.data._id)
      .then(user => {
        var response;
(....)
        delete response._source._id;

        return  kuzzle.pluginsManager.trigger('auth:getCurrentUser', response);
      })
      .then(response => {
        return q(new ResponseObject(requestObject, response));
      });
  };